client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
 	const oldPackage = newPackages[ packageId ];
 
+	const oldBox = state.form.packages.all[ oldPackage.box_id ];
+	const newBox = state.form.packages.all[ boxTypeId ];
+	const weight =
```

Comment:
@dechov I wanted to circle back to this issue. We do expect to have weights that are less than 1 [unit] of measurement, e.g. 0.2lbs, etc.

The questions for our code here are:
1. How many decimal places do we need for the weight measurement?
2. Will we need to use a different method for adding the weights together, rather than the currently-used floating-point addition option?